.github/workflows/build-push-image-main.yml

```diff
@@ -0,0 +1,30 @@
+name: Tutor Build Action Test
+
+on:
+  push:
+    branches:
+      - 'pearson-release/olive.main'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get current time
+        uses: josStorer/get-current-time@v2
+        id: current-time
+        with:
+          format: YYYY-MM-DD
```

Comment:
Is this package safe to use? Does it have any support? Did you try to create a function instead of using an external package? @alexjmpb